linuxkm/linuxkm_wc_port.h

```diff
@@ -1412,6 +1414,8 @@
             return 0;
         }
     #else
+        #define WC_MUTEX_OPS_INLINE
```

Comment:
This is defined in both the if and the else? Can be done just once above the if.